File: python/6.py
def sumOfSquares(max: int) -> int:
    sum = 0

    for i in range(1, max+1):
        sum += i**2

    return sum

def squareOfSum(max: int) -> int:
    sum = 0

    for i in range(1, max+1):
        sum += i
    
    return sum ** 2

print(abs(squareOfSum(100) - sumOfSquares(100)))

File: typescript/6.ts
function sumOfSquares(max: number): number {
  let sum = 0;

  for (let i = 1; i <= max; i++) {
    sum += i ** 2;
  }

  return sum;
}

function squareOfSum(max: number): number {
  let sum = 0;

  for (let i = 1; i <= max; i++) {
    sum += i;
  }

  return sum ** 2;
}

console.log(Math.abs(squareOfSum(100) - sumOfSquares(100)));
// 25164150

export {};
